apps/web/pages/api/v1/clerk/webhooks.ts

```diff
@@ -0,0 +1,64 @@
+import type { IncomingHttpHeaders } from "http";
+import type { NextApiRequest, NextApiResponse } from "next";
+import type { WebhookRequiredHeaders } from "svix";
+import type { WebhookEvent } from "@clerk/nextjs/server"
+import { Webhook } from "svix";
+
+
+const webhookSecret: string | undefined = process.env.WEBHOOK_SECRET;
```

Comment:
Can you add this to lib/env.ts and rename with a prefix, we already got a stripe webhook secret and I'd like to clearly see what's what

```suggestion
const webhookSecret: string | undefined = env.CLERK_WEBHOOK_SECRET
```